front/src/modules/Items/ItemsPage.tsx

```diff
@@ -46,7 +46,7 @@ export default function ItemsScreen() {
       },
     });
     if (response.result === "success") {
-      setList(response.list);
+      setList(response.list.slice(0, 20));
```

Comment:
This doesn't seem right

Reply:
Sorry i forgotten delete this  i will fix it